### PR TITLE
[BUG FIX] [MER-3887] Take author to new workspace after login

### PR DESCRIPTION
**Pull request by nicocirio**
[Link](https://eliterate.atlassian.net/browse/MER-3887) to the ticket

### Before

https://github.com/user-attachments/assets/7ba5633c-a97b-430f-b9cd-d59c7dc3d432

### After

https://github.com/user-attachments/assets/846d5a0c-e7f7-4dad-95fd-5a89b49f4f31

### CODE REVIEW

**Review by simonchoxx — APPROVED** [review]
LGTM 🚀

File: lib/oli_web/controllers/pow/session_html/new.html.heex [hunk 1]
```diff
@@ -42,7 +42,7 @@
           >
             OR
           </div>
-          <%= form_for @changeset, @action, [as: :user], fn f -> %>
+          <%= form_for @changeset, Routes.authoring_pow_session_path(OliWeb.Endpoint, :create, request_path: ~p"/workspaces/course_author"), [as: :user], fn f -> %>
```
Comment (eliknebel) [inline lib/oli_web/controllers/pow/session_html/new.html.heex:45]: This works and is a good solution for this login, however I think the real culprit here is the routes defined in `lib/oli_web/pow/author_routes.ex` for `after_sign_in_path` and `after_registration_path`

**Review by eliknebel — CHANGES_REQUESTED** [review]

**Comment by eliknebel** [comment]
I think there are a few other places that could possibly lead a user into the old views:
- lib/oli_web/controllers/project_plugs.ex
- lib/oli_web/plugs/authorize_project.ex
- lib/oli_web/plugs/community_admin.ex

Replacing the references to OliWeb.Projects.ProjectsLive in these views with the new workspace route should fix it

**Comment by simonchoxx** [comment]
@eliknebel Ok, I will change the references to `OliWeb.Projects.ProjectsLive` in those other three places to go to `/workspaces/course_author`

File: lib/oli_web/controllers/pow/session_html/new.html.heex (hunk 1 above)
Comment (simonchoxx) [reply lib/oli_web/controllers/pow/session_html/new.html.heex:45]: @eliknebel Do you think we should keep the code as it was and modify the routes in those two functions (after_sign_in_path and after_registration_path) ???

**Review by manelli — APPROVED** [review]

**Review by eliknebel — APPROVED** [review]

**Pull request merged**